ceph.spec.in

```diff
@@ -67,7 +67,8 @@ Release:	@RPM_RELEASE@%{?dist}
 Epoch:		2
 %endif
 
-# define %_epoch_prefix macro which will expand to the empty string if %epoch is undefined
+# define %%_epoch_prefix macro which will expand to the empty string if
+# %%epoch is undefined
```

Comment:
@ktdreyer Would the comment be more readable if it read:

```
# define an "_epoch_prefix" macro which will expand to the empty string if
# the "epoch" macro is undefined?
```

(Frankly, I find the double `%` aethetically displeasing.)

If not, then I'm OK to merge it as it is.